### ISSUE TITLE
View - Create Search Input with Form Validation

### ISSUE THREAD

**Issue by lexoskeleton** [lexoskeleton/feq_dictionary_app#4]
We need an HTML input search bar that does not allow blank searches.

**Comment by audreyclarkdev** [comment]
Created a pull request today for this issue. Right after that, I saw that there's an "icon-search.svg" in the images. We should probably use that instead of Font Awesome. Do you have any thoughts?

**Comment by audreyclarkdev** [comment]
Also, should I marked this closed once the pull request is merged into dev, @lexoskeleton?

**Comment by knopgm** [comment]
Fix- fix the path for the "icon-search.svg" image.
 src="./assets/images/icon-search.svg"      instead of       src="/start-here/assets/images/icon-search.svg"

``
